bundles/org.openhab.binding.lgwebos/README.md

```diff
@@ -62,6 +62,33 @@ Thing lgwebos:WebOSTV:tv1 [host="192.168.2.119", key="6ef1dff6c7c936c8dc5056fc85
 | mediaPlayer     | Player    | Media control player                                                                                                                                                                                                    | W          |
 | mediaStop       | Switch    | Media control stop                                                                                                                                                                                                      | W          |
 | appLauncher     | String    | Application ID of currently running application. This also allows to start applications on the TV by sending a specific Application ID to this channel.                                                                 | RW         |
+| rcButton        | String    | Simulates pressing of a button on the TV's infrared remote control. See below for a list of button names.                                                                                                               | W          |
```

Comment:
still mentions "infrared". Would leave it out. Just mentione ... on the TV's remote control.